specificatie/filter.yaml

```diff
@@ -26,5 +26,5 @@ components:
       description: |
         Een naam die door de gemeente aan een openbare ruimte is gegeven. **Gebruik van de wildcard is toegestaan. Zie [feature-beschrijving](https://github.com/VNG-Realisatie/Haal-Centraal-common/blob/v1.3.0/features/wildcard.feature)** **Zoeken met tekstvelden is [case-Insensitive](https://github.com/VNG-Realisatie/Haal-Centraal-BRP-bevragen/blob/v1.3.0/features/case_insensitive.feature).
       type: string
-      pattern: ^(\?+|\*)?[a-zA-Z0-9À-ÿ '\-\']{7,80}(\?+|\*)?$
+      pattern: ^(\?+|\*)?[a-zA-Z0-9À-ÿ \-\']{7,80}(\?+|\*)?$
```

Comment:
deze is nog niet goed, want alleen bij gebruik van wildcard mag minimum 7 zijn. Anders kan je korte straatnaam zoals "spui" niet zoeken